doc/asciidoc/consumer/index.adoc

```diff
@@ -55,6 +62,85 @@ MERGE (n:Label {id: event.id})
     ON CREATE SET n += event.properties
 ----
 
+==== Change Data Capture Event
+
+This method allows to ingest CDC events coming from another Neo4j Instance. You can use two strategies:
+
+ * The `Merge` strategy which merges the nodes/relationships by the CDC event `id` field (it's related to the Neo4j physical ID
```

Comment:
SourceId strategy, .... using the default label, and id .xx which can also be configured